src/features/venue/atoms/VenueType.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+import { VenueTypeCode } from 'api/gen'
+import { mapTypeToIcon } from 'libs/parsers'
+
+import { IconWithCaption } from './IconWithCaption'
+
+interface VenueTypeProps {
```

Comment:
Il n'y pas d'utilité à nommer les props d'une manière spécifique puisque celles-ci ne sont pas réutilisées : 
```suggestion
interface Props {
```